scripts/assembly_status_tracking/Manual_results_email.sh

```diff
@@ -0,0 +1,73 @@
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#!/usr/bin/bash
+
+## move to appropriate dir
+cd $PWD
+
+## Source private setup:
+source ./ensembl-production-metazoa-private/conf/_assembly_status.conf
+
+# #Source ENV to run release RR tracker
+source $ASM_VENV
+
+# Codon Cluster path
+export PATH=$PATH:$MYSQL_SHORT_CMDS
+
+## Recipient email list
+ENS_METAZOA_CONTACTS="lcampbell@ebi.ac.uk ensembl-metazoa@ebi.ac.uk sdyer@ebi.ac.uk"
+ENS_VERTS_CONTACTS="lcampbell@ebi.ac.uk leanne@ebi.ac.uk ensembl-genebuild@ebi.ac.uk"
+ENS_PLANTS_CONTACTS="lcampbell@ebi.ac.uk gnaamati@ebi.ac.uk shradha@ebi.ac.uk sdyer@ebi.ac.uk jalvarez@ebi.ac.uk"
+ENS_FUNGI_CONTACTS="lcampbell@ebi.ac.uk nishadi@ebi.ac.uk mcarbajo@ebi.ac.uk"
+ENS_PROTISTS_CONTACTS="lcampbell@ebi.ac.uk nishadi@ebi.ac.uk mcarbajo@ebi.ac.uk"
```

Comment:
For easier maintenance and given the current scenario, I would simplify the list of emails:
```suggestion
ENS_METAZOA_CONTACTS="ensembl-metazoa@ebi.ac.uk"
ENS_VERTS_CONTACTS="ensembl-genebuild@ebi.ac.uk"
ENS_PLANTS_CONTACTS="ensembl-plants@ebi.ac.uk"
ENS_FUNGI_CONTACTS="mcarbajo@ebi.ac.uk"
ENS_PROTISTS_CONTACTS="mcarbajo@ebi.ac.uk"
```